scripts/ansible/oe-linux-acc-setup.yml

```diff
@@ -22,6 +22,7 @@
     - import_role:
         name: linux/az-dcap-client
         tasks_from: stable-install.yml
+      when: ansible_distribution_version != "20.04"
```

Comment:
Is this because the `az-dcap-client` isn't available for 20.04 yet?

Reply:
Yes, that is correct. This line is to be removed when the package is available so the final production-ready image can be created.